cmake/modules/dts.cmake

```diff
@@ -86,7 +86,7 @@ find_package(Dtc 1.4.6)
 #   C preprocessor when generating the devicetree from DTS_SOURCE
 # - DTS_SOURCE: the devicetree source file to use may be pre-set
 #   with this variable; otherwise, it defaults to
-#   ${BOARD_DIR}/${BOARD}.dts
+#   ${BOARD_DIRECTORIES}/${BOARD}.dts
```

Comment:
nit: `${BOARD}.DTS` is a HWMv1 leftover. Maybe now is a good time to update it?

Reply:
Fixed